releasenotes/notes/fix-adder-gates-39cf3d5f683e8880.yaml

```diff
@@ -0,0 +1,19 @@
+---
+fixes:
+  - |
+    Added default definitions for :class:`.FullAdderGate`, :class:`.HalfAdderGate`,
+    :class:`.ModularAdderGate` and :class:`.MultiplierGate` gates, allowing to 
+    contruct :class:`qiskit.quantum_info.Operator`\s from quantum circuits 
```

Comment:
```suggestion
    contruct :class:`.Operator`\s from quantum circuits 
```

Reply:
Thanks! Done in [c39d08e](https://github.com/Qiskit/qiskit/pull/13530/commits/c39d08e0f6f91b1d80ddead3f15078b46219670a).